Equilobe.DailyReport.Models/Storage/GeneratedSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Equilobe.DailyReport.Models.Storage
+{
+    public class GeneratedSettings
+    {
+        public long ReportSettingsId { get; set; }
+        public DateTime LastReportSentDate { get; set; }
+        public DateTime LastDraftSentDate { get; set; }
+        public bool WasResetToDefaultToday { get; set; }
```

Comment:
Pentru ca am facut modificarile din bool in data pentru confirmare, cred ca proprietatea asta, si operatia de reset in sine, nu stiu daca mai e necesara. Sau oricum nu in forma curenta.